prog/vm/github_runner.rb

```diff
@@ -327,7 +327,12 @@ def setup_info
       COMMAND
     end
 
-    if !github_runner.label.include?("ubuntu")
+    if github_runner.label.include?("gpu")
+      message = "The GPU runners will be deprecated on December 31, 2025. All jobs using these runners should be migrated to other runner types."
```

Comment:
How visible is this error message to the user? Does GitHub alert the user about the error, is it visible on the main summary page, or is it only shown when looking at job details/logs?